src/frontend/src/instance/distributed.rs

```diff
@@ -273,19 +272,14 @@ impl DistInstance {
                 self.drop_table(table_name).await
             }
             Statement::Insert(insert) => {
-                let (catalog, schema, _) =
-                    table_idents_to_full_name(insert.table_name(), query_ctx.clone())
-                        .map_err(BoxedError::new)
-                        .context(error::ExternalSnafu)?;
-
                 let insert_request =
                     SqlHandler::insert_to_request(self.catalog_manager.clone(), &insert, query_ctx)
```

Comment:
Maybe next PR we can directly convert `insert`into row format.

Reply:
Start immediately